docs/plans/006-self-review-tracer-bullet.html

```
@@ -0,0 +1,151 @@
+<pre>## 1. No regression
+./scripts/verify.sh            # green
+
+## 2. Real self-review posts (operator-gated, frozen-head PR)
+cargo run --locked -- review-pr \
+  --number &lt;PR&gt; --repo misty-step/cerberus \
+  --model openrouter/&lt;glm-5.2-slug&gt; \
+  --allow-env OPENROUTER_API_KEY \
+  --timeout-seconds 1800 --summary-target status \
+  --out-dir target/cerberus/self-review --dry-run   # READ the artifact first
+# then re-run with --post
+# exit 0; artifact passes ReviewArtifact.v1; PR shows a Cerberus summary + >=1 inline comment on a changed line.
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)